bot/seasons/evergreen/trivia_quiz.py

```diff
@@ -45,71 +48,66 @@ def load_questions() -> dict:
             questions = json.load(json_data)
             return questions
 
-    @commands.command(name="quiz", aliases=["trivia"])
+    @cooldown(1, 20, BucketType.channel)
```

Comment:
This `cooldown` also prevents moderators from stopping a quiz as long as it's active. I'm not even sure if the user who started the quiz should be unable to stop it due to a cooldown, but moderators should certainly be able to stop it without having to wait for the cooldown to finish.

I think this ties into this command method being one monolithic callable that tries to perform multiple actions, including both starting and stopping the quiz. A separate command to stop the quiz, like `!quiz stop` would solve that.

Reply:
Well, it would not make sense to stop the game right after starting it, so that was my plan for adding the cooldown.
I switched to using the single command instead of a command group having 2 commands `start` and `stop` as it was suggested to me.
I can add it back or would it be better to just remove the cooldown?